pkg/operator/controller/clientca-configmap/controller_test.go

```diff
@@ -0,0 +1,304 @@
+				ic(false, "ca-bundle"),
+				cm("openshift-config", "ca-bundle", expectedData),
+				cm("openshift-ingress", "router-client-ca-test", expectedData),
+			},
+			expectCreate: []client.Object{},
+			expectUpdate: []client.Object{
+				ic(false, "ca-bundle", "ingresscontroller.operator.openshift.io/finalizer-clientca-configmap"),
+			},
+			expectDelete: []client.Object{},
+		},
+		{
+			name: "source configmap present, target configmap present but inconsistent, finalizer present",
+			existingObjects: []runtime.Object{
+				ic(false, "ca-bundle", "ingresscontroller.operator.openshift.io/finalizer-clientca-configmap"),
+				cm("openshift-config", "ca-bundle", expectedData),
+				cm("openshift-ingress", "router-client-ca-test", unexpectedData),
+			},
+			expectCreate: []client.Object{},
+			expectUpdate: []client.Object{
+				cm("openshift-ingress", "router-client-ca-test", expectedData),
+			},
+			expectDelete: []client.Object{},
+		},
+		{
+			name: "source configmap present, target configmap absent, finalizer absent, ingresscontroller deleted",
+			existingObjects: []runtime.Object{
+				ic(true, "ca-bundle"),
+				cm("openshift-config", "ca-bundle", expectedData),
+			},
+			expectCreate: []client.Object{},
+			expectUpdate: []client.Object{},
+			expectDelete: []client.Object{},
+		},
+		{
+			name: "source configmap present, target configmap present, finalizer absent, ingresscontroller deleted",
+			existingObjects: []runtime.Object{
+				ic(true, "ca-bundle"),
+				cm("openshift-config", "ca-bundle", expectedData),
+				cm("openshift-ingress", "router-client-ca-test", expectedData),
+			},
+			expectCreate: []client.Object{},
+			expectUpdate: []client.Object{},
+			expectDelete: []client.Object{
+				cm("openshift-ingress", "router-client-ca-test", expectedData),
+			},
+		},
+		{
+			name: "source configmap present, target configmap present but inconsistent, finalizer present, ingresscontroller deleted",
+			existingObjects: []runtime.Object{
+				ic(true, "ca-bundle", "ingresscontroller.operator.openshift.io/finalizer-clientca-configmap"),
+				cm("openshift-config", "ca-bundle", expectedData),
+				cm("openshift-ingress", "router-client-ca-test", unexpectedData),
+			},
+			expectCreate: []client.Object{},
+			expectUpdate: []client.Object{
+				ic(true, "ca-bundle"),
+			},
+			expectDelete: []client.Object{
+				cm("openshift-ingress", "router-client-ca-test", unexpectedData),
+			},
+		},
+	}
+
+	scheme := runtime.NewScheme()
+	operatorv1.Install(scheme)
+	corev1.AddToScheme(scheme)
+
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			fakeClient := fake.NewClientBuilder().
+				WithScheme(scheme).
+				WithRuntimeObjects(tc.existingObjects...).
+				Build()
+			cl := &fakeClientRecorder{fakeClient, t, []client.Object{}, []client.Object{}, []client.Object{}}
+			reconciler := &reconciler{
+				client: cl,
+				config: Config{
+					OperatorNamespace: "openshift-ingress-operator",
```

Comment:
As far as the tests are concerned, it seems we could drop setting the `OperatorNamespace`.

Reply:
I pushed https://github.com/openshift/cluster-ingress-operator/pull/948/commits/3a3fb20502db4bb46db3de1ffef89faa5f84cc6a to remove the field.